js/custom-post-parents-admin.js

```diff
@@ -1,61 +1,45 @@
+
 ;(function ($) {
     $(function () {
         var $ul = $('.custom-post-parents-selectedlist');
-        var $adders = $('.custom-post-parents-selectlist-add');
-
-        $ul.on('click', '.ntdelbutton', function () {
+        $ul.on('click','.ntdelbutton',function(){
+            var id_option=$(this).attr("name");
+            var section=$(this).attr("data-section");
+            $("#cpt-f-"+section+" >option[id='"+id_option+"']").show();
             $(this).parents('span').remove();
```

Comment:
Hay muchos usos del $(this), en estos casos es mejor crear una variable
var $this= $(this);
Y usarla para no ir creando instancias en cada llamada